source/manuals/access-aws-accounts.html.md.erb

```diff
@@ -0,0 +1,154 @@
+---
+title: Accessing AWS Accounts
+account_list:
+  - team: 'GOV.UK'
+    url: 'https://github.com/alphagov/govuk-aws-data/blob/master/docs/govuk-aws-accounts.md'
+  - team: 'Verify'
+    url: 'https://github.com/alphagov/verify-blackbox-passwords/blob/master/aws-accounts.info'
+  - team: 'Digital Marketplace'
+    url: 'https://alphagov.github.io/digitalmarketplace-manual/aws-accounts.html#available-roles'
+---
+
+# <%= current_page.data.title %>
+
+GDS manage a number of Amazon Web Services (AWS) accounts.
+User accounts are managed centrally in a "base account" from which users
+can access the target accounts through a process called "assuming roles".
+
+## Adding new users to GDS' AWS accounts
+
+Currently new joiners have no access to AWS by default. To grant a user access
+to AWS they need to be added to the `gds-users` base account. Once a user is in
+the base account it is up to the owners of the target accounts to grant the
+user access to roles within the account.
+
+To add a user to the `gds-users` base account fill in [this form](https://gds-request-an-aws-account.cloudapps.digital/).
```

Comment:
we generally do reference-style markdown links in this repo. That is, links like this:

```
You can access [foo][] by doing steps [bar and baz][alternate-id].

[foo]: https://example.com/foo
[alternate-id]: https://example.com/doing-steps-bar-and-baz
```

Aside from being consistent, I find this helps with editing the text without the links getting in the way of reading whole sentences.